flexmeasures/ui/templates/views/status.html

```diff
@@ -55,6 +58,99 @@ <h3>Data connectivity for sensors of {{ asset.name }}</h3>
                         {% endfor %}
                     </tbody>
                 </table>
+                <h3>Latest scheduling jobs of {{ asset.name }}</h3>
+                <table id="scheduling_jobs" class="table table-striped table-responsive paginate nav-on-click">
+                    <thead>
+                        <tr>
+                            <th>Job id</th>
+                            <th>Asset id</th>
+                            <th>Sensor or asset</th>
+                            <th>Created at</th>
+                            <th class="text-right no-sort">Status</th>
+                            <th class="hidden">URL</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for job_data in scheduling_job_data: %}
+                        <tr title="View data">
+                            <td>
+                                {{ job_data.job_id }}
+                            </td>
+                            <td>
+                                {{ job_data.asset_id }}
+                            </td>
+                            <td>
+                                {{ job_data.asset_or_sensor_type }}
+                            </td>
+                            <td>
+                                {{ job_data.enqueued_at | naturalized_datetime }}
```

Comment:
One more thing: I believe the exact time of queueing the job might matter to get the right one, so let's show the exact datetime string in the title, so the user can see it on hover.

Reply:
Nevermind, I added it so we can merge!